feature/staff/src/main/java/io/github/droidkaigi/confsched2020/staff/ui/item/StaffItem.kt

```diff
@@ -9,13 +15,29 @@ import io.github.droidkaigi.confsched2020.staff.R
 import io.github.droidkaigi.confsched2020.staff.databinding.ItemStaffBinding
 
 class StaffItem @AssistedInject constructor(
-    @Assisted private val staff: Staff
+    @Assisted private val staff: Staff,
+    context: Context,
+    private val lifecycleOwnerLiveData: LiveData<LifecycleOwner>
 ) : BindableItem<ItemStaffBinding>(staff.id.hashCode().toLong()),
     EqualableContentsProvider {
+
+    private val placeHolder = VectorDrawableCompat.create(
```

Comment:
Thank you!
We can use lazyWithParam like this. So you don't have to context for the constructor 👍 

```kotlin
    private val layoutInflater by lazyWithParam<Context, LayoutInflater> { context ->
        LayoutInflater.from(context)
    }
```